keyboards/funky40/keymaps/v1.1/keymap.c

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2020 /u/TheFourthCow
```

Comment:
Keyboards **must** have a `default` keymap, which my suggestion is to rename the `v1.1` folder to `default`